.github/workflows/fast_forward.yaml

```diff
@@ -0,0 +1,42 @@
+name: Fast forward latest release
+
+on:
+  push:
+    branches:
+    - main
```

Comment:
Is there any easy way to toggle this off when there is a feature code freeze?

Reply:
Yes. Disable it. 😄 But really (and I don't know why this didn't occur to me before), we can probably just go ahead and cut the next version at that time? If there's no downstream to pick up the subsequent version, it seems like it'd be safe to do that?